### PR TITLE
Update /openstack/install page

### LINKED ISSUE #6320: [COPY UPDATE] install - "Install Ubuntu OpenStack on a cluster of servers"
Please update install - "Install Ubuntu OpenStack on a cluster of servers" from the [copy doc](https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by solazio**
## Done

Update /openstack/install page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/install
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy doc](https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc)

## Issue / Card

Fixes #6320

### CODE REVIEW

**Comment by barrymcgee** [comment]
@solazio Can you resolve the items you addressed please?

**Comment by solazio** [comment]
> @solazio Can you resolve the items you addressed please?

@barrymcgee done

File: templates/openstack/install.html [hunk 1]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+        <h4 class="p-stepped-list__title">Bootstrap and configure other nodes</h4>
         <div class="p-stepped-list__content">
-          <p>conjure-up offers two options with OpenStack:</p>
-          <p>1. OpenStack with NovaLXD</p>
-          <p>This installs OpenStack configured to use the LXC &lsquo;machine container&rsquo; hypervisor &mdash; containers that behave like ultra-fast VMs. Choose this option to run Linux workloads in LXD containers, for higher density and for evaluation of OpenStack.</p>
-          <p>2. OpenStack with NovaKVM</p>
-          <p>This installs OpenStack configured to use the KVM hypervisor ready to launch full virtual machines. We recommend this option for production clouds.</p>
+          <p>All other nodes should be configured to PXE boot from the provisioning subnet
+          by default.</p>
+          <p>Once configured, power them on and wait until they show up in the ‘Machines’
```
Comment (sowasred2012) [inline templates/openstack/install.html]: Double quotes are used for label names throughout the rest of these instructions:

```suggestion
          <p>Once configured, power them on and wait until they show up in the &ldquo;Machines&ldquo;
```

File: templates/openstack/install.html [hunk 2]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+          </div>
+          <p>Add MAAS cloud by running:</p>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" value="juju add-cloud --local"
+              readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>Answer the questions accordingly:</p>
+          <pre><code>Select cloud type: <strong>maas</strong>
+            Enter a name for your maas cloud: <strong>maas</strong>
```
Comment (sowasred2012) [inline templates/openstack/install.html]:
```suggestion
Enter a name for your maas cloud: <strong>maas</strong>
```

File: templates/openstack/install.html [hunk 3]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+            Enter the API endpoint url: <strong>http://172.16.7.2:5240/MAAS</strong>
```
Comment (sowasred2012) [inline templates/openstack/install.html]:
```suggestion
Enter the API endpoint url: <strong>http://172.16.7.2:5240/MAAS</strong>
```

File: templates/openstack/install.html [hunk 4]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+            Cloud "maas" successfully added to your local client.
```
Comment (sowasred2012) [inline templates/openstack/install.html]:
```suggestion
Cloud "maas" successfully added to your local client.</code></pre>
```

File: templates/openstack/install.html [hunk 5]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+          </code></pre>
```
Comment (sowasred2012) [inline templates/openstack/install.html]: These changes will remove unnecessary white space.
```suggestion
```

File: templates/openstack/install.html [hunk 6]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+            <input class="p-code-copyable__input" value="juju add-credential maas"
+              readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>And answer the questions accordingly. Use the MAAS API key in the maas-oauth
+          line:</p>
+          <pre><code>Do you ONLY want to add a credential to this client? (Y/n): <strong>Y</strong>
+Enter credential name: <strong>maas</strong>
+Regions
+default
```
Comment (sowasred2012) [inline templates/openstack/install.html]:
```suggestion
  default
```

File: templates/openstack/install.html [hunk 7]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+          </div>
+          <p>This command takes a while. Once finished you should see the following output
+          on the bottom:</p>
+          <pre><code>Deploy of bundle completed.</code></pre>
+          <p>But it does not mean that your OpenStack deployment is done. In fact, it has
+          just started. You have to monitor the status by running juju status command.
+          The deployment is completed once all applications, units and machines turn to
+          the &ldquo;active&rdquo; state as depicted on the listing below:</p>
+          <pre><code>$ juju status
+Model Controller Cloud/Region Version SLA Timestamp
```
Comment (sowasred2012) [inline templates/openstack/install.html]: White space formatting has been lost here, making the output look very noisy - lines 800 to 885 should be replaced with:

```
Model      Controller       Cloud/Region  Version  SLA          Timestamp
openstack  maas-controller  maas/default  2.7.0    unsupported  12:22:38-05:00

App                    Version  Status  Scale  Charm                  Store       Rev  OS      Notes
ceph-mon               14.2.2   active      3  ceph-mon               jujucharms   44  ubuntu  
ceph-osd               14.2.2   active      3  ceph-osd               jujucharms  294  ubuntu  
ceph-radosgw           14.2.2   active      1  ceph-radosgw           jujucharms  283  ubuntu  
cinder                 15.0.0   active      1  cinder                 jujucharms  297  ubuntu  
cinder-ceph            15.0.0   active      1  cinder-ceph            jujucharms  251  ubuntu  
glance                 19.0.0   active      1  glance                 jujucharms  291  ubuntu  
keystone               16.0.0   active      1  keystone               jujucharms  309  ubuntu  
mysql                  5.7.20   active      1  percona-cluster        jujucharms  281  ubuntu  
neutron-api            15.0.0   active      1  neutron-api            jujucharms  282  ubuntu  
neutron-gateway        15.0.0   active      1  neutron-gateway        jujucharms  276  ubuntu  
neutron-openvswitch    15.0.0   active      3  neutron-openvswitch    jujucharms  269  ubuntu  
nova-cloud-controller  20.0.0   active      1  nova-cloud-controller  jujucharms  339  ubuntu  
nova-compute           20.0.0   active      3  nova-compute           jujucharms  309  ubuntu  
ntp                    3.2      active      4  ntp                    jujucharms   36  ubuntu  
openstack-dashboard    16.0.0   active      1  openstack-dashboard    jujucharms  297  ubuntu  
placement              2.0.0    active      1  placement              jujucharms    1  ubuntu  
rabbitmq-server        3.6.10   active      1  rabbitmq-server        jujucharms   97  ubuntu  

Unit                      Workload  Agent  Machine  Public address  Ports              Message
ceph-mon/0                active    idle   1/lxd/0  172.16.7.164                       Unit is ready and clustered
ceph-mon/1                active    idle   2/lxd/1  172.16.7.163                       Unit is ready and clustered
ceph-mon/2*               active    idle   3/lxd/0  172.16.7.165                       Unit is ready and clustered
ceph-osd/0*               active    idle   1        172.16.7.160                       Unit is ready (1 OSD)
ceph-osd/1                active    idle   2        172.16.7.161                       Unit is ready (1 OSD)
ceph-osd/2                active    idle   3        172.16.7.162                       Unit is ready (1 OSD)
ceph-radosgw/0*           active    idle   0/lxd/0  172.16.7.166    80/tcp             Unit is ready
cinder/0*                 active    idle   1/lxd/2  172.16.7.168    8776/tcp           Unit is ready
  cinder-ceph/0*          active    idle            172.16.7.168                       Unit is ready
glance/0*                 active    idle   2/lxd/1  172.16.7.167    9292/tcp           Unit is ready
keystone/0*               active    idle   3/lxd/2  172.16.7.169    5000/tcp           Unit is ready
mysql/0*                  active    idle   0/lxd/2  172.16.7.170    3306/tcp           Unit is ready
neutron-api/0*            active    idle   1/lxd/2  172.16.7.171    9696/tcp           Unit is ready
neutron-gateway/0*        active    idle   0        172.16.7.159                       Unit is ready
  ntp/0*                  active    idle            172.16.7.159    123/udp            chrony: Ready
nova-cloud-controller/0*  active    idle   2/lxd/2  172.16.7.172    8774/tcp,8775/tcp  Unit is ready
nova-compute/0*           active    idle   1        172.16.7.160                       Unit is ready
  neutron-openvswitch/2   active    idle            172.16.7.160                       Unit is ready
  ntp/3                   active    idle            172.16.7.160    123/udp            chrony: Ready
nova-compute/1            active    idle   2        172.16.7.161                       Unit is ready
  neutron-openvswitch/1   active    idle            172.16.7.161                       Unit is ready
  ntp/2                   active    idle            172.16.7.161    123/udp            chrony: Ready
nova-compute/2            active    idle   3        172.16.7.162                       Unit is ready
  neutron-openvswitch/0*  active    idle            172.16.7.162                       Unit is ready
  ntp/1                   active    idle            172.16.7.162    123/udp            chrony: Ready
openstack-dashboard/0*    active    idle   3/lxd/2  172.16.7.174    80/tcp,443/tcp     Unit is ready
placement/0*              active    idle   2/lxd/3  172.16.7.175    8778/tcp           Unit is ready
rabbitmq-server/0*        active    idle   0/lxd/2  172.16.7.176    5672/tcp           Unit is ready

Machine  State    DNS           Inst id              Series  AZ       Message
0        started  172.16.7.159  openstack1           bionic  default  Deployed
0/lxd/0  started  172.16.7.166  juju-592de2-0-lxd-3  bionic  default  Container started
0/lxd/1  started  172.16.7.170  juju-592de2-0-lxd-4  bionic  default  Container started
0/lxd/2  started  172.16.7.176  juju-592de2-0-lxd-5  bionic  default  Container started
1        started  172.16.7.160  openstack2           bionic  default  Deployed
1/lxd/0  started  172.16.7.164  juju-592de2-1-lxd-3  bionic  default  Container started
1/lxd/1  started  172.16.7.168  juju-592de2-1-lxd-4  bionic  default  Container started
1/lxd/2  started  172.16.7.171  juju-592de2-1-lxd-5  bionic  default  Container started
2        started  172.16.7.161  openstack3           bionic  default  Deployed
2/lxd/0  started  172.16.7.163  juju-592de2-2-lxd-4  bionic  default  Container started
2/lxd/1  started  172.16.7.167  juju-592de2-2-lxd-5  bionic  default  Container started
2/lxd/2  started  172.16.7.172  juju-592de2-2-lxd-6  bionic  default  Container started
2/lxd/3  started  172.16.7.175  juju-592de2-2-lxd-7  bionic  default  Container started
3        started  172.16.7.162  openstack4           bionic  default  Deployed
3/lxd/0  started  172.16.7.165  juju-592de2-3-lxd-3  bionic  default  Container started
3/lxd/1  started  172.16.7.169  juju-592de2-3-lxd-4  bionic  default  Container started
3/lxd/2  started  172.16.7.174  juju-592de2-3-lxd-5  bionic  default  Container started
```

File: templates/openstack/install.html [hunk 8]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+started
+3 started 172.16.7.162 openstack4 bionic default Deployed
+3/lxd/0 started 172.16.7.165 juju-592de2-3-lxd-3 bionic default Container
+started
+3/lxd/1 started 172.16.7.169 juju-592de2-3-lxd-4 bionic default Container
+started
+3/lxd/2 started 172.16.7.174 juju-592de2-3-lxd-5 bionic default Container
+started
+          </code></pre>
+          <p>This is what you should be able to see in the &lsquo;Machines&rsquo; tab in MAAS:</p>
```
Comment (sowasred2012) [inline templates/openstack/install.html]: Single vs double quotes inconsistency:
```suggestion
          <p>This is what you should be able to see in the &ldquo;Machines&ldquo; tab in MAAS:</p>
```

File: templates/openstack/install.html [hunk 9]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>The Horizon service is available at the IP address displayed by running:</p>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input"
+              value="juju status | grep openstack-dashboard | tail -n 1 | awk '{print $5}'" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>Go to <code>http://&lt;Horizon IP&gt;:80/horizon</code> URL. You should see the
+          OpenStack login screen. Type your credentials and press the ‘Sign In’ button. Use
```
Comment (sowasred2012) [inline templates/openstack/install.html]:
```suggestion
          OpenStack login screen. Type your credentials and press the &ldquo;Sign In&ldquo; button. Use
```

File: templates/openstack/install.html [hunk 10]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+          ‘admin_domain’ in the ‘Domain’ field:</p>
```
Comment (sowasred2012) [inline templates/openstack/install.html]:
```suggestion
          &ldquo;admin_domain&ldquo; in the &ldquo;Domain&ldquo; field:</p>
```

File: templates/openstack/install.html [hunk 11]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
           <p>
-            <a href="https://assets.ubuntu.com/v1/ff3cfa6b-build-openstack-with-conjure-up-5.png">
-              <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/ff3cfa6b-build-openstack-with-conjure-up-5.png?w=424" width="424" alt="User preferences for admin interface on MAAS" />
+            <a href="https://assets.ubuntu.com/v1/88a77f0d-openstack+dashboard+login.png">
+              <img src="https://assets.ubuntu.com/v1/88a77f0d-openstack+dashboard+login.png?w=848"
+                width="424" height="241" alt="" />
             </a>
           </p>
-          <p>Next, you will have the opportunity to configure the individual OpenStack services.</p>
+          <p>You should be able to see the OpenStack welcome screen. Click on the ‘admin’
```
Comment (sowasred2012) [inline templates/openstack/install.html]:
```suggestion
          <p>You should be able to see the OpenStack welcome screen. Click on the &ldquo;admin&ldquo;
```

File: templates/openstack/install.html [hunk 12]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+            drop-down list in the top right corner and press ‘OpenStack RC File’:</p>
```
Comment (sowasred2012) [inline templates/openstack/install.html]:
```suggestion
            drop-down list in the top right corner and press &ldquo;OpenStack RC File&ldquo;:</p>
```

File: templates/openstack/install.html [hunk 13]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
           </p>
-        </div>
-      </li>
-
-      <li class="p-stepped-list__item">
-        <h4 class="p-stepped-list__title"><span class="p-stepped-list__bullet">11</span>Configure and deploy the OpenStack services</h4>
-        <div class="p-stepped-list__content">
-          <p>Choose <strong>&lsquo;Configure&rsquo;</strong> to change the default configuration of any component. When all components are configured to your liking, choose <strong>&lsquo;Deploy&rsquo;</strong>.</p>
+          <p>Some browsers may require that you confirm that you want to download this file.
+          If you are using Google Chrome, press ‘Keep’ from the menu on the bottom:</p>
```
Comment (sowasred2012) [inline templates/openstack/install.html]:
```suggestion
          If you are using Google Chrome, press &ldquo;Keep&ldquo; from the menu on the bottom:</p>
```

File: templates/openstack/install.html [hunk 14]
```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
...
+|           |              |   public: http://172.16.7.167:9292                                       |
+|           |              | RegionOne                                                                |
+|           |              |   internal: http://172.16.7.167:9292                                     |
+|           |              | RegionOne                                                                |
+|           |              |   admin: http://172.16.7.167:9292                                        |
+|           |              |                                                                          |
++-----------+--------------+--------------------------------------------------------------------------+
+          </code></pre>
+          <p>In order to create tenants, networks, launch instances, etc., refer to the
+          python-openstackclient <a href="https://docs.openstack.org/python-openstackclient/latest/cli/command-list.html">documentation</a>.</p>
```
Comment (sowasred2012) [inline templates/openstack/install.html]: This should use `p-link--external`

File: templates/openstack/install.html [hunk 15]
```diff
@@ -435,7 +1039,7 @@ <h2>Need more help?</h2>
   </div>
 </div>
 
-{% with first_item="_cloud_bootstack", second_item="_download_cloud_buy_landscape", third_item="_download_documentation" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
+{% with first_item="_cloud_bootstack", second_item="_download_cloud_buy_landscape", third_item="_openstack_documentation" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
```
[the remark this answers is not in the document]
Comment (solazio) [reply templates/openstack/install.html]: @pmahnke can you please confirm if I should modify [`_cloud_bootstack` include](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/templates/shared/contextual_footers/_cloud_bootstack.html) or I should create a new one with the content they want [here]()https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc/edit#?

**Pull request merged**